docs/getting-started/dev-docker-setup.md

```diff
@@ -50,11 +50,22 @@ has_toc: true
 5. Build application python environment and web assets (make sure that you use node 14; check by running `node -v` and switch if you need to by running `nvm use 14`)
   ```sh
   invenio-cli packages lock --pre --dev
+  ```
+8.5. Due to a [known bug in the current version of InvenioRDM](https://github.com/inveniosoftware/invenio-files-rest/issues/264) we can only use `setuptools` version smaller then 58.
```

Comment:
I think these section-numbers are just wrong, right? I believe this one should be `6` instead of... `8.5`? 🤷🏻‍♂️ , and continues from there

Reply:
some of my new steps got lost in the merge conflict after ekate's reversion. good catch! should be fixed now: https://github.com/nyudlts/ultraviolet/blob/e32abab92f039d059b61e45f6a0c7c31745b92d8/docs/getting-started/dev-docker-setup.md